opengever/wopi/discovery.py

```diff
@@ -49,9 +53,14 @@ def run_discovery(net_zone='external-https', url=None):
     ):
         return
 
-    resp = requests.get(url)
-    tree = ET.XML(resp.text)
-    wopi_discovery = etree_to_dict(tree)['wopi-discovery']
+    try:
+        resp = requests.get(url)
+        tree = ET.XML(resp.text)
+        wopi_discovery = etree_to_dict(tree)['wopi-discovery']
+    except Exception as exc:
+        logger.error('WOPI discovery from %r failed:' % url)
+        logger.exception(exc)
```

Comment:
`logger.exception()` automatically adds the exception info to the logging message.

```suggestion
        logger.exception('WOPI discovery from %r failed:', url)
```